hack/validate-release-config.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Whats the advantage of this script over directly calling the release-controller? It just passes through one arg

Reply:
Updated to call release-controller directly